doozerlib/brew.py

```diff
@@ -81,6 +81,65 @@ def watch_task(brew_hub, log_f, task_id, terminate_event):
     return error
 
 
+def watch_tasks(brew_hub, log_f, task_ids, terminate_event):
+    if not task_ids:
+        return
+    end = time.time() + 4 * 60 * 60
+    session = koji.ClientSession(brew_hub, opts={'serverca': '/etc/pki/brew/legacy.crt'})
```

Comment:
this caught my eye. the whole reason for adding this oddly-specific CA was because we didn't understand how to handle `certifi` before. i don't think we need it any more and would rather not see it peppered around the code.

but more importantly, `watch_task` was the only method here that crafted its own client connection instead of accepting one as a parameter, and this is just a multiplexed copy of that. is there a good reason for wanting a bare client session? it seems to me that what's calling `watch_tasks(self.runtime.group_config.urls.brewhub, ...` would do better to call it with `self.runtime.build_retrying_koji_client()`.

Reply:
Right, I just copied from `watch_task` then modified on top of it. Better to use the `retrying_koji_client`